spec.bs

```diff
@@ -646,14 +646,22 @@ To <dfn>construct a pending fenced frame config</dfn> given an [=auction config=
 To <dfn>fill in a pending fenced frame config</dfn> given a [=fenced frame config=]
 |pendingConfig|, [=auction config=] |auctionConfig|, and [=leading bid info=] |winningBidInfo|:
 1. Let |winningBid| be |winningBidInfo|'s [=leading bid info/leading bid=].
-1. Set |pendingConfig|'s [=fenced frame config/mapped url=]'s [=mapped url/value=] to
-   |winningBid|'s [=generated bid/ad descriptor=]'s [=ad descriptor/url=].
 1. Let |adSize| be |winningBid|'s [=generated bid/ad descriptor=]'s [=ad descriptor/size=].
+1. Let |concreteAdSize| be the result of TODO: resolving screen-relative sizes to pixel values for

```

Comment:
Can you maybe file a follow-up bug for this task? It's not clear exactly what work will need to go into this, however I know various specs handle things like pixel density and screen sizes, so a tracked issue might be a good place to have discussion about how exactly to do this. Then we can link to it from here.

Reply:
(Still relevant)

Reply:
https://github.com/WICG/turtledove/issues/986